phpstan.neon.dist

```diff
@@ -0,0 +1,17 @@
+includes:
+  - vendor-bin/phpstan/baseline.neon
+
+parameters:
+  # Enforce minimal supported version (note: 7.0 can't be set in PHPStan)
+  phpVersion: 70100
```

Comment:
There's no point in running PHPStan multiple times on multiple PHP versions if 7.1 is forced in the configuration like this.

Reply:
Thanks for the tip, I didn't know it. I just wanted to point PHP version since it can't be automatically determined from `composer.json` because PHPStan is not installed as project's dev dependency.

Reply:
@ondrejmirtes I've just tested PHPStan on current branch's state (green level 1) but with other PHP versions (8.0.20 and 8.1.7) and I got 23 errors, internal errors included (`Internal error: Internal error: syntax error, unexpected token "match" in file test/Util/DocblockTest.php` - wtf, there isn't even `match` in this file 😅). So it looks like there is difference between PHP/deps versions.

@bobthecow there's also problem that PHPStan does not allow to configure 7.0 here and also [`phpstan/phpstan-phpunit`](https://packagist.org/packages/phpstan/phpstan-phpunit) requires 7.2... So it really would help if PsySH supported only 7.4+ 🙂

Reply:
Just run SA once on for example 8.1 (if it's supported) with the right deps and don't bother with anything more complicated.